instrumentation/database/sql/splunksql/README.md

```diff
@@ -0,0 +1,18 @@
+# Splunk Instrumentation for the `database/sql` Package
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/signalfx/splunk-otel-go/instrumentation/database/sql/splunksql.svg)](https://pkg.go.dev/github.com/signalfx/splunk-otel-go/instrumentation/database/sql/splunksql)
+
+This package instruments the [`database/sql`](https://pkg.go.dev/database/sql)
+package.
+
+## Getting Started
+
+This package is designed to be used in conjunction with existing database
+drivers that have been instrumented so OpenTelemetry semantic conventions can
+be discovered. An example of this type of use can be found
+[here](../../../github.com/go-sql-driver/mysql/splunkmysql).
+
+This package can be used directly as well. It becomes the users responsibility
+to ensure accurate and complete information about the database system is passed
+as attributes to ensure OpenTelemetry semantic conventions are satisfied. An
+example of this can be found [here](./example_test.go).
```

Comment:
what about `./example_sql_test.go`? I think it would be good to mention it or combine those two examples.

Reply:
The `example_sql_test.go` file contains info on how to register an `InstrumentationConfig`. It is rendered in the go docs:

![20210927_104626](https://user-images.githubusercontent.com/5543599/134959171-98286c3d-67d0-4582-a255-0a7ce27423e3.png)

Having it structured this way that example is put into the appropriate context for the user looking for information about how to use the `Register` function. In the paragraph above I link to the complete example from the `splunksql` package of how to setup instrumentation this way. I can add something in that paragraph about the other example file, but I'm not sure it is going to be the best in the long term. Having the Go docs handle this seems sufficient. What do you think @pellared?

Reply:
We can also link to the pkg.go.dev docs from the README. What do you think of that?

Reply:
I think what is now should be good enough. If someone will ask us for help in future it would mean that the docs should be improved.